src/sdk/object_sdk.js

```diff
@@ -656,6 +655,10 @@ class ObjectSDK {
                 dbg.warn(`upload_object with copy_sources - copying by reading source first (not server side)
                 so it can take some time and cause client timeouts`);
             }
+            if (params.tagging_copy) {
```

Comment:
doesn't moving this to here effect line 612: 
```if (params.tagging_copy) params.tagging = source_md.tagging;```

Reply:
this line only affects copying between noobaa buckets, as other buckets copy within the server is handled by the SDK. in the noobaa bucket case the tagging is already returned by read_object_md